### DISCUSSION TITLE
Adding prompt tags via Python SDK (or other code based interactions)

### DISCUSSION THREAD

**Discussion by simon-hiel** [langfuse/langfuse#2072] [Ideas]
### Describe the feature or potential improvement

**Context**: Trying to migrate 100s of prompts from my database to langfuse. 

**Request**: I don't want to manually add for each prompt the relevant tags - instead when creating the prompt via the SDK I want to be able to add tags to the prompt.
```python

 langfuse.create_prompt(
        name="dummy name",
        prompt="dummy prompt",
        labels=["production"],
        # tags=["add","tags","here"]
    )

```

**Comment by clemra** [comment]
Thank you, Simon. That makes a lot of sense. We will document this in the SDKs and APIs. Will get back to you with an ETA.

**Comment by marcklingen** [comment]
This is great, just turned this into an issue and will try to add this soonish to APIs and SDKs: https://github.com/langfuse/langfuse/issues/2075
